docs/execute/binaries.md

```diff
@@ -28,7 +28,7 @@ There are two ways for running the C++ library tests with a custom tracer:
 
 ## Golang library
 
-1. To test unmerged PRs locally, run the following in the utils/build/docker/golang/parametric directory:
+To test unmerged PRs locally, run the following in the utils/build/docker/golang/parametric directory:
```

Comment:
The examples below don't show how to run an unmerged commit (although hard to give an example without a real commit hash 😆 ), and that's exactly what I was confused about when I first ran these tests.
Maybe let's change line 31 to be more descriptive:
```suggestion
To test unmerged PRs locally, you'll first need to identify a git commit hash that contains your changes. If your changes have not yet been pushed to a remote branch, you can run `git log` locally and capture the latest (or most relevant) commit hash. If your changes are pushed to github, you can copy the latest (or most relevant) commit hash from the github UI on your open PR.
Once you have the commit hash, run the following in the utils/build/docker/golang/parametric directory:
```